db/redis.js

```diff
@@ -53,6 +53,16 @@ async function addDoorTestSensorData(request, response){
     response.status(200).json("OK")
 }
 
+const addIM21DoorSensorData = (locationid, doorSignal) => {
```

Comment:
It looks like the majority of the functions in this file are using the style:
```
function functionName(params) {
...
}
```

instead of 
```
const functionName = (params) => {
...
}
```

Could you please use that instead for this function and `addVitals`?